Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,12 @@
 import React from 'react';
 import './Header.css';
+
+function HeaderTitle() {
+    return (
+        <div className="HeaderTitle">
+            <span id="UserName">Lambda School</span><span class="faded" id="UserHandle">@Lambda School</span><span class="faded" id="date">11 february</span>
```

Comment:
It'll become natural to use className instead of class.

```suggestion
            <span id="UserName">Lambda School</span><span className="faded" id="UserHandle">@Lambda School</span><span className="faded" id="date">11 february</span>
```